src/dropdown/Dropdown.js

```diff
@@ -9,22 +10,16 @@ const themes = {
 };
 
 const PropTypes = {
-  children: React.PropTypes.oneOfType([
-    React.PropTypes.node,
-    React.PropTypes.element,
-    React.PropTypes.array
-  ]),
-  theme: React.PropTypes.oneOf(['semantic']),
-  valueLink: React.PropTypes.shape({
-    value: React.PropTypes.string,
-    requestChange: React.PropTypes.func
-  })
+  children: t.ReactNode,
```

Comment:
Mmm no it's a mistake thanks (you can use `options` as this is a wrapper of `react-select`)